ci/build_container/build_container.sh

```diff
@@ -8,7 +8,7 @@ apt-get install -y wget software-properties-common make cmake git python python-
   bc libtool automake zip time golang g++ gdb strace
 # clang head (currently 5.0)
 wget -O - http://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
-apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial main"
+apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-5.0 main"
```

Comment:
Probably because they just forked and are getting ready for 5.0 release and main will switch to 6.0. I would suggest that we stick with 5.0 for now.